.github/workflows/main.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     name: ChatGPT explain code # Job name
     steps:
```

Comment:
ChatGPT Review:<br/>
    
    The code in this Pull Request is responsible for using the ChatGPT Github Actions from cirolini repository. This allows for the use of the OpenAI API key and the Github Token from repository secrets. There are no bugs in the code.